client/src/components/NearestHospital/HospitalSidebar.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import styled from "styled-components";
+
+import { SearchBar } from 'antd-mobile';
+import { NavLink } from "react-router-dom";
+import LocalEmergencyNo from './LocalEmergencyNo';
+
+
+const HospitalSidebar = props => {
+
+const SearchBarContainer = styled.div`
+   padding: 5rem 0;
+`;
+
+const NavBar = styled.div`
+   border-top: 0.2px solid rgba(0, 0, 0, 0.5);
+   border-bottom: 0.2px solid rgba(0, 0, 0, 0.5);
+   ul {
+     list-style-type: none;
+     padding: 1rem 0;
+     li {
+        margin: 2rem 0;
+           a {
+             text-decoration: none;
+             transition: all .4s;
+             color: inherit;
+             padding-left: 2.5rem;
+           }
+           a:hover {
+             font-weight: bold;
+           }
+       }
+     }
+
+   }
+`;
+
+
+const StyledSearchBar = styled(SearchBar)`
+  &.am-search {
+    background-color: #fff;
+    border: 0.1rem solid rgba(0, 0, 0, 0.5);
+    border-radius: 4rem;
+  }
+  .am-search-cancel-show {
+    display: none;
+   }
+`;
+
+const ActiveLinkStyles = {
+  fontWeight: "bold",
+  borderLeft: "4px solid #425af2",
+  paddingLeft: "2rem"
+}
+
+
+  const [searchValue, setSearchValue] = useState('');
+
+  const onSearchInputChange = (value) => {
+    setSearchValue(value);
+    console.log(searchValue);
+
+  }
+  const clearSearch = () => {
+    setSearchValue('');
+  }
+
+
+
+return (
+   <div>
+     <SearchBarContainer>
+          <StyledSearchBar
+            value={searchValue}
+            placeholder=""
+            onClear={clearSearch}
+            onChange={onSearchInputChange}
+           />
+     </SearchBarContainer>
+     <NavBar>
+        <ul>
+           <li><NavLink activeStyle={ActiveLinkStyles} to="/nearest-hospital">Health Facilities</NavLink></li>
```

Comment:
On mobile needs to match designs, also related to #293 

current
![Screen Shot 2020-05-12 at 12 04 34 AM](https://user-images.githubusercontent.com/42228386/81648847-376df100-93e4-11ea-9da3-6b6cf5a7cae9.png)

figma
![Screen Shot 2020-05-12 at 12 04 46 AM](https://user-images.githubusercontent.com/42228386/81648853-3a68e180-93e4-11ea-84c7-6e92d34c1640.png)

Reply:
thanks for your review. Doesn't this mean i will have to wait for issue #293 to completed or take on the issue myself? because i was a little bit confused there was no existing layout for the mobile view being that feed has already been designed.

Reply:
> thanks for your review. Doesn't this mean i will have to wait for issue #293 to completed or take on the issue myself? because i was a little bit confused there was no existing layout for the mobile view being that feed has already been designed.

I think we can link this PR to that issue because you've already started on it here . 

I also see this in the Feed so maybe this component should be rendered at a more global level. For now we can have here but we may need to refactor it in the future.